ruby/dalli/test/performance/browsing_test.rb

```diff
@@ -1,12 +1,19 @@
 require 'test_helper'
 require 'rails/performance_test_help'
 
-class BrowsingTest < ActionDispatch::PerformanceTest
-  # Refer to the documentation for all available options
-  # self.profile_options = { :runs => 5, :metrics => [:wall_time, :memory]
-  #                          :output => 'tmp/performance', :formats => [:flat] }
-
-  def test_homepage
-    get '/'
-  end
-end
+# Currently broken in 0.0.7 of rails-perftest.
```

Comment:
These tests don't work anymore, because `rails-perftest` has a bug in it that hasn't been fixed. We can enable this again whenever they get around to fixing it.